src/picknik_ur_site_config/objectives/open_door_fixed_handle_ml.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="Open Door Fixed Handle ML">
+  <!-- ////////// -->
+  <BehaviorTree ID="Open Door Fixed Handle ML" _description="Attempts to open a door with a fixed handle (such as cabinet doors) using ML." _favorite="true" _hardcoded="false">
+    <Control ID="Sequence" name="TopLevelSequence">
+      <!-- Load the objectives needed for this behavior -->
+      <Action ID="LoadObjectiveParameters" config_file_name="config_open_door_fixed_handle_MTC.yaml" parameters="{parameters}"/>
+      <!-- Update the admittance controller and reset the FTS -->
+      <SubTree ID="Re-Zero Force-Torque Sensors" _collapsed="true"/>
+      <SubTree ID="Update Admittance Controller" _collapsed="true"/>
+      <!-- Look at the desired location where objects (door) will be present -->
+      <Action ID="MoveToWaypoint" waypoint_name="Right Wall" planning_group_name="manipulator" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" use_all_planners="true" constraints=""/>
+      <SubTree ID="GetDoorGraspableObjectSubtree" _collapsed="true"/>
+
+      <!-- Get the masks for the handle subframe of the door GraspableObject -->
+      <Control ID="Fallback">
+        <Control ID="Sequence">
+          <Action ID="MoveToWaypoint" waypoint_name="Right Wall" planning_group_name="manipulator" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" use_all_planners="false" constraints=""/>
+          <SubTree ID="GetHandleGraspableObjectSubtree" _collapsed="true"/>
+        </Control>
+        <Control ID="Sequence">
+          <Action ID="MoveToWaypoint" waypoint_name="Extended Right" planning_group_name="manipulator" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" use_all_planners="false" constraints=""/>
+          <SubTree ID="GetHandleGraspableObjectSubtree" _collapsed="true"/>
+        </Control>
+      </Control>
```

Comment:
You could consider making these a bit more concise in the top-level BT by putting the `MoveToWaypoint` Behavior also inside the subtree, and having the waypoint name another remapped port of the subtree (all other parameters just hard-coded inside the subtree)

Reply:
This is largely for ease of use developing. I think of the Get<Object>Subtrees as running from wherever they currently are so poses can be tuned on the fly or a pose generator for inspections could be used.

Reply:
Yeah, maybe I was just playing "XML golf" on this one, agree that you don't necessarily need to move to a waypoint before executing the object detection subtree.